packages/contentful--react-apps-toolkit/package.json

```
@@ -31,7 +31,9 @@
     "commit-msg": "commitlint -E HUSKY_GIT_PARAMS"
   },
   "peerDependencies": {
-    "react": ">=16.8.0"
+    "react": ">=16.8.0",
+    "@contentful/app-sdk": ">=4.3.5",
+    "@contentful/f36-components": ">=4.0.0"
```

I think this is better, right?

Also, why make f36 a peer dep?

```suggestion
    "@contentful/app-sdk": "^4.3.5",
    "@contentful/f36-components": "^4.0.0"
```

---

Can we just depend on `@contentful/f36-spinner`? We don't need the full repo. This would also make the bundle smaller or the treeshaking easier.

---

I would expect that f36 is more likely to be already installed in the app anyway already, that's why I choose the whole library. 
I honestly think we could also get rid of the spinner and just add a loading string or just return null, wdyt?

---

Not having forma 36 as a dependency would be great, yes. We could make it an optional dependency and if it's installed, we render the spinner?

`@contentful/f36-spinner` is a dependency `@contentful/f36-components`, so I'd always just directly use that to reduce the node_modules folder. Some users might not want to install all of forma 36.

---

Good idea, I've added it now as an optional dependent and render nothing if it is not available